pkg/codegen/inner.tmpl

```diff
@@ -1,6 +1,8 @@
 package {{.Package}}
 
 import (
+    "strings"
```

Comment:
Yes it's lame to have to do this. On the other hand I never know what `goimports` is gonna do, it scares me.